src/JLD00.jl

```diff
@@ -541,15 +541,15 @@ function write{T<:Union{HDF5BitsKind, String}}(parent::Union{JldFile, JldGroup},
         close(dtype)
     end
 end
-write{T<:Union{HDF5BitsKind, String}}(parent::Union{JldFile, JldGroup}, name::String, data::Union{T, Array{T}}) =
+write(parent::Union{JldFile, JldGroup}, name::String, data::Union{T, Array{T}}) where {T<:Union{HDF5BitsKind, String}} =
     write(parent, name, data, full_typename(typeof(data)))
 
 # Arrays-of-arrays of basic types
-write{T<:Union{HDF5BitsKind, String}}(parent::Union{JldFile, JldGroup}, name::String,
-                                            data::Array{Array{T,1}}, astype::String) =
+write(parent::Union{JldFile, JldGroup}, name::String,
+                                            data::Array{Array{T,1}}, astype::String) where {T<:Union{HDF5BitsKind, String}} =
```

Comment:
@Keno It would be nice if it could unindent lines as appropriate to match up the brackets vertically after removing the old parametric syntax. Otherwise lines like this look really weird.